### PR TITLE
Add unit to dashbaords_generator & mark some yt_dashboards with unit

### PR DESCRIPTION
**Pull request by jasminetealover**
Added `unit` parameter to `cell` for clearer and more intuitive data visualization in Grafana dashboards. Also, marked some already existed dashboards with it.

### CODE REVIEW

File: yt/admin/dashboards/yt_dashboards/artemis.py [hunk 1]
```diff
@@ -597,17 +612,21 @@ def build_changelogs():
...
                 "Data node Split changelogs bytes rate",
-                changelog("split_changelogs.bytes.rate"))
+                changelog("split_changelogs.bytes.rate"),
+                unit="bytes")
         .row()
             .cell(
                 "Data node Multiplexed changelogs records rate",
-                changelog("multiplexed_changelogs.records.rate"))
+                changelog("multiplexed_changelogs.records.rate"),
+                unit="bytes")
```
Comment (Kontakter) [inline yt/admin/dashboards/yt_dashboards/artemis.py:625]: It doesn't look like 'bytes'

File: yt/admin/dashboards/yt_dashboards/artemis.py [hunk 2]
```diff
@@ -597,17 +612,21 @@ def build_changelogs():
...
             .cell(
                 "Data node Split changelogs records rate",
-                changelog("split_changelogs.records.rate"))
+                changelog("split_changelogs.records.rate"),
```
Comment (Kontakter) [inline yt/admin/dashboards/yt_dashboards/artemis.py:628]: This neither

File: yt/admin/dashboards/yt_dashboards/artemis.py [hunk 3]
```diff
@@ -915,7 +946,7 @@ def build_local_artemis():
         d.add_parameter(
             "pod",
             "Pod",
-            GrafanaTextboxDashboardParameter("tnd-0"))
+            GrafanaTextboxDashboardParameter("tnd-teapot-0"))
```
Comment (Kontakter) [inline yt/admin/dashboards/yt_dashboards/artemis.py:949]: Why this change needed?

Comment (jasminetealover) [reply yt/admin/dashboards/yt_dashboards/artemis.py:949]: oops. my bad

**Review by Kontakter — APPROVED** [review]

**Comment by Kontakter** [comment]
I have read some code around, and there is a similar implementation for monitoring: `SystemFields.Unit` tag and method `unit` in `class Taggable`

Could we reuse it here? Or at least implement `.unit_grafana(...)` and `.unit_monitoting(...)` (since the unit system of grafana and monitoring differ)

**Comment by jasminetealover** [comment]
Closing in favor of https://github.com/ytsaurus/ytsaurus/pull/924
